docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,374 @@
+offline installation and walks you through the different potential scenarios.
+
+## Prepare for Deployment
+
+In preparation for the offline deployment you will download the Canonical
+Kubernetes snap, fulfill the networking requirements based on your scenario and
+handle images for workloads and Canonical Kubernetes features.
+
+### Prep 1: Download the Canonical Kubernetes snap
+
+From a machine with access to the internet download the following:
+
+```bash
+sudo snap download k8s --channel 1.30-classic/beta
+sudo snap download core20
+sudo mv k8s_*.snap k8s.snap
+sudo mv k8s_*.assert k8s.assert
+sudo mv core20_*.snap core20.snap
+sudo mv core20_*.assert core20.assert
+```
+
+The [core20][Core20] and `k8s` snap are downloaded. The `core20.assert` and 
+`k8s.assert` files, are necessary to verify the integrity of the snap packages.
+
+```{note} Update the version of k8s by adjusting the channel parameter.
+    Find the version you desire in the [snapstore](https://snapcraft.io/k8s).```
+
+```{note} With updates to the snap the base core is subject to change.```
+
+### Prep 2: Network Requirements
+
+Air-gap deployments are typically associated with a number of constraints and
+restrictions with the networking connectivity of the machines.
+Verify that your cluster nodes can communicate, machines have a default gateway
+and optionally ensure proxy access.
+
+#### Network Requirement: Cluster node communication
+<!-- TODO: Services and Ports Doc -->
+Ensure that all cluster nodes are reachable from each other.
+Refer to [Services and ports][svc-ports] used for a list of all network ports
+used by Canonical Kubernetes.
+
+#### Network Requirement: Default Gateway
+
+Kubernetes services use the default network interface of the machine
+for the means of node discovery:
+
+- kube-apiserver (part of kubelite) 
+  - uses the default network interface to advertise this address to 
+    other nodes in the cluster.
+  - Without a default route kube-apiserver does not start.
+- kubelet (part of kubelite)
+  - uses the default network interface to pick the node's InternalIP address.
+  - A default gateway greatly simplifies the process of setting up the
+    network feature.
+
+In case your air gap environment does not have a default gateway,
+you can add a dummy default route on interface eth0 using the following command:
+
+```bash
+ip route add default dev eth0
+```
+
+```{note} Confirm the name of your default network interface used for pod-to-pod
+    communication by running "ip a".
+```
+
+```{note} The dummy gateway will only be used by the Kubernetes services to 
+    know which interface to use,
+    actual connectivity to the internet is not required.
+    Ensure that the dummy gateway rule survives a node reboot.
+```
+
+#### (Optional) Network Requirement: Ensure proxy access
+
+Please skip this section, if you can't use an HTTP proxy (e.g. squid)
+with limited access to 
+image registries (e.g. docker.io, quay.io, rocks.canonical.com, etc).
```

Comment:
"This is only required if access to upstream image registries (...) is only allowed through an HTTP proxy (e.g. squid)"